lib/bb/generate_report_request_form.rb

```diff
@@ -7,10 +7,10 @@ class GenerateReportRequestForm < Common::Form
                                 familyhealthhistory seifoodjournal healthcareproviders healthinsurance
                                 seiimmunizations labsandtests medicalevents medications militaryhealthhistory
                                 seimygoalscurrent seimygoalscompleted treatmentfacilities
-                                vitalsandreadings prescriptions medications vaallergies
+                                vitalsandreadings prescriptions vaallergies
                                 vaadmissionsanddischarges futureappointments pastappointments
                                 vademographics vaekg vaimmunizations vachemlabs vaprogressnotes
-                                vapathology vaproblemlist varadiology vahth wellness dodmilitaryservice ).freeze
+                                vapathology vaproblemlist varadiology vahth wellness dodmilitaryservice ).uniq.freeze
```

Comment:
should we be calling `uniq` in the `EligibleDataClasses` model by defining a new accessor method for `data_classes`?